src/lib.rs

```diff
@@ -162,7 +162,8 @@ impl Plugin for OpenXrPlugin {
             let mut manual_texture_views = app.world.resource_mut::<ManualTextureViews>();
             manual_texture_views.insert(LEFT_XR_TEXTURE_HANDLE, left);
             manual_texture_views.insert(RIGHT_XR_TEXTURE_HANDLE, right);
-            drop(manual_texture_views);
+            //drop non drop
+            //drop(manual_texture_views);
```

Comment:
this was a change I wasn't sure of

Reply:
yea this seems fine to just remove. Probably leftover from an older bevy version where this was `Drop`